contributors/devel/running-locally.md

```diff
@@ -92,6 +95,30 @@ This will build and start a lightweight local cluster, consisting of a master an
 You can use the `./cluster/kubectl.sh` script to interact with the local cluster. `./hack/local-up-cluster.sh` will
 print the commands to run to point kubectl at the local cluster.
 
+### Using Kind
+kind is a tool for running local Kubernetes clusters using Docker container "nodes". kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI. For more information please check [kind](https://github.com/kubernetes-sigs/kind).
+
+To create a cluster from Kubernetes source:
+- ensure that `kind` is [installed](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- ensure that Kubernetes is cloned in `$(go env GOPATH)/src/k8s.io/kubernetes`
+- build a node image and create a cluster with:
+
+```sh
+kind build node-image
+kind create cluster --image kindest/node:latest
+```
+
+**Note:** If the Kubernetes is cloned in path other than `$(go env GOPATH)/src/k8s.io/kubernetes` you can build node-image by specifying --kube-root args with value as path to the kubernetes source.
```

Comment:
This is covered in more detail in the kind getting started guide currently. We should probably spin it out into a guide for developing kubernetes with kind.

Reply:
Sure  will open a separate PR for documenting this separately!